.jshintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "eqnull": true,
+  "unused": "vars",
+  "bitwise": true,
+  "eqeqeq": true,
+  "es3": true,
```

Comment:
After discussion with @justinvdm, we've decided to bump this to `es5`. This will also let use change our `preset` to [airbnb](https://github.com/jscs-dev/node-jscs/blob/master/presets/airbnb.json), which is (and I quote): "nice."

We need to be careful of IE and trailing commas, though.